boards/arm/google_kukui/doc/index.rst

```diff
@@ -0,0 +1,73 @@
+.. _google_kukui_board:
+
+Google Kukui EC
```

Comment:
Since this board is not announced yet, is it OK to be publishing this information?

Reply:
Yes, "Kukui" is a publicly available codename